scripts/build/Platform/Linux/build_config.json

```diff
@@ -43,7 +43,7 @@
     "steps": [
       "profile_nounity",
       "asset_profile_nounity",
-      "test_profile_nounity"
+      "test_profile_nounity_arm64"
```

Comment:
Can we just use unity in this case?

The normal Ubuntu x86_64 build already covers the non-unity build

Reply:
In general, we use the 'nounity' builds for daily/weekly builds. I think switching to 'unity' will introduce additional metrics that may not be intended for this
```
  "profile": {
    "TAGS": [
      "periodic-incremental-daily",
      "periodic-clean-weekly-internal",
      "daily-pipeline-metrics",
      "weekly-build-metrics"
    ],
```
vs
```
  "profile_nounity": {
    "TAGS": [
      "daily-pipeline-metrics",
      "weekly-build-metrics"
    ],

```

Reply:
Why can't we ad another tag.

Reply:
we could if its important

Reply:
My suggestion was purely for saving some build time and build resources.